appveyor.yml

```diff
@@ -14,12 +17,14 @@ install:
   - SET OPENSSL_ROOT_DIR=C:\tools
   - SET PATH=C:\tools\bin;%PATH%
 
+  - ps: wget 'https://s3.amazonaws.com/kylo-pl-bucket/curl-7.42.1-x86_64_mingw-w64_4.8.3_win32_seh.7z' -OutFile "$pwd\curl-7.42.1-x86_64_mingw-w64_4.8.3_win32_seh.7z"
+  - ps: 7z.exe x "curl-7.42.1-x86_64_mingw-w64_4.8.3_win32_seh.7z" -oC:\tools | FIND /V "ing "
```

Comment:
Yeah, it's needed by the leatherman 0.3.5 package we're using; its `leatherman/lib/cmake/leatherman/LeathermanConfig.cmake` includes ```
leatherman_component(curl)``` which was breaking the build. See https://ci.appveyor.com/project/puppetlabs/cpp-pcp-client/build/1.0.116.

I was planning to get this in and then make a ticket to fix this useless dependency.
Do you have an alternative to fix? Perahps, @branan or @MikaelSmith, I'm doing something wrong.